### ISSUE TITLE
Append some text to a MultiLineEntry

### ISSUE THREAD

**Issue by dennyvsdev** [fyne-io/fyne#1653]
I think appending to TextArea or ... is common in other languages (for example showing logs of some application). I looked for a widget or function to append in MultiLineEntry or a similar solution in other widgets but couldn't find any. So I asked in StackOverflow: [Append some text to a MultiLineEntry in fyne](https://stackoverflow.com/q/65276566/10019712) and found some alternative way a "trick" to do so using @raoyc answer:

```go
largeText := widget.NewMultiLineEntry()
largeText.SetText("Lorem ipsum ...")
originalText := largeText.Text
fmt.Println(originalText)
newText := originalText + "appending new text"
largeText.SetText(newText)
```

I think using the same solution then creating a function for NewMultiLineEntry widget can be like:

```go
largeText := widget.NewMultiLineEntry()
largeText.AppendText("Lorem ipsum ...")
```

Also noting that the overall performance is too slow and I might post another issue with stats and how slow it is in low-end pc. However, Keep up the good work. Fyne is the best with all its flaws comparing to other GUI frameworks.

**Comment by Jacalz** [comment]
Hi @dennyvsdev. It is great to hear that you enjoy using fyne. I think this issue might be a duplicate of https://github.com/fyne-io/fyne/issues/1631?

**Comment by Jacalz** [comment]
As a suggestion, I think you could simplify your code to avoid copying the large text twice. This should hopefully make things a bit faster for you (at least for large amounts of text) 🙂 
```go
largeText := widget.NewMultiLineEntry()
largeText.SetText("Lorem ipsum ...")
largeText.SetText(largeText.Text + "appending new text")
```

**Comment by andydotxyz** [comment]
I'm sure that we could support something like this.
However, if we add AppendText then do we need PrependText, InsertText, RemoveText etc as well?

That said, until we have the internal text display optimisations they will not be any more performant than manipulating the text and setting new content, as you already did - which I think is why the reporter closed #1631.
